lib/prediction/predictor.ts

```diff
@@ -99,6 +103,10 @@ class Worker extends events.EventEmitter {
             args.push('--embeddings', process.env.GENIENLP_EMBEDDINGS);
         if (process.env.GENIENLP_DATABASE)
             args.push('--database', process.env.GENIENLP_DATABASE);
+        if (fs.existsSync(this._modeldir + '/' + CALIBRATOR_FILE_NAME)) {
```

Comment:
I changed genienlp so that it reads mc_dropout from the saved calibrator, and the calibrator path defaults to `$model_path/calibrator.pkl`. So no need to pass anything here.